ansible/playbooks/roles/upgrade/tasks/opensearch/migrate-odfe.yml

```diff
@@ -0,0 +1,226 @@
+---
+- name: OpenSearch | Get information about installed packages as facts
+  package_facts:
+    manager: auto
+  when: ansible_facts.packages is undefined
+
+- name: OpenSearch | Print ElasticSearch ond OpenSearch versions
+  debug:
+    msg:
+      - "Elasticsearch version currently installed: {{ ansible_facts.packages['elasticsearch-oss'][0].version }}"
+      - "Opensearch version to be installed: {{ opensearch_defaults.file_name_version.opensearch[ansible_architecture].split('-')[1] }}"
+
+- name: ODFE migr | Ensure elasticsearch cluster is up and running
```

Comment:
Maybe ODFE migration?

Reply:
ok
Rename accordingly to your wish now.